src/components/TransactionCellOracleRespondTx.vue

```diff
@@ -16,15 +18,16 @@
 import AppChip from '@/components/AppChip'
 import TransactionArrowRightIcon from '@/components/TransactionArrowRightIcon'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
-import { formatDecodeByteArray, formatEllipseHash } from '@/utils/format'
+import { formatDecodeByteArray } from '@/utils/format'
 
 const props = defineProps({
   transactionData: {
     required: true,
     type: Object,
   },
 })
-const oracleResponse = computed(() =>
-  formatEllipseHash(formatDecodeByteArray(props.transactionData.response)),
-)
+const oracleResponse = computed(() => {
+  const decodedResponse = formatDecodeByteArray(props.transactionData.response)
```

Comment:
```suggestion
  const { response } = props.transactionData
  const decodedResponse = response instanceof Array ? formatDecodeByteArray(response) : response.toString()
```

This should also handle empty cases, what do you think?

Reply:
Ok, good. I wanted to avoid fixing data inconsistencies on FE, but ok, I accept your suggestion. It's a bit better solution
However, it didn't solve it ultimately. There are still some characters that cannot be printed, looking like it's 'empty'

![image](https://github.com/aeternity/aescan/assets/15363559/410fcc8c-5296-44a8-b864-0be8565e5066)